docs/en/Running-Flink-on-Alluxio.md

```diff
@@ -48,8 +48,9 @@ the directory usually ends with `etc/hadoop`.)
 ### Generate and Distribute the Alluxio Client Jar
 
 In order to communicate with Alluxio, we need to provide Flink programs with the Alluxio Core Client
-jar.
-Generate the Flink compatible client jar by building the entire project from the top level `alluxio`
+jar. If you are using Alluxio binary, then compilation is unnecessary and the jar is available at
```

Comment:
They shouldn't need to rebuild the jar - we generate a flink client at ${ALLUXIO_HOME}/client/flink/alluxio-${{ALLUXIO_RELEASED_VERSION}}-flink-client.jar

Reply:
The primary audience for docs is the people using a distribution. I think the footnote "Alternatively, advanced users can choose ..." below is good enough to help those who want to build from source

Reply:
Done.